### DISCUSSION TITLE
Requests: Filter Between TV Shows and Movies

### DISCUSSION THREAD

**Discussion by war59312** [sct/overseerr#3217] [Feature Requests]
I like to have a filter for Requests for TV Shows and Movies.

**Comment by Silvan-M** [comment]
Yea or like described [here](https://github.com/sct/overseerr/discussions/2506) to filter for movie genre, duration, etc.
Would really be helpful searching for movies.

EDIT: And [here](https://github.com/sct/overseerr/discussions/2697).
